doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -23,6 +23,56 @@ reference these objects is not allowed.
+| `error_message`           | [`text`]                     | The error message, if the statement failed.                                                                                                                                                                                                                                   |
+| `rows_returned`           | [`bigint`]                   | The number of rows returned, for statements that return rows.                                                                                                                                                                                                                 |
+| `execution_strategy`      | [`text`]                     | For `SELECT` queries, the strategy for executing the query. `constant` means computed in the control plane without the involvement of a cluster, `fast-path` means read by a cluster directly from an in-memory index, and `standard` means computed by a temporary dataflow. |
+| `transaction_id`          | [`uint8`]                    | The ID of the transaction that the statement was part of. Note that transaction IDs are only unique per session.                                                                                                                                                              |
+| `prepared_statement_id`   | [`uuid`]                     | An ID that is unique for each prepared statement. For example, if a statement is prepared once and then executed multiple times, all executions will have the same value for this column (but different values for `execution_id`).                                           |
+| `sql`                     | [`text`]                     | The SQL text of the statement.                                                                                                                                                                                                                                                |
+| `prepared_statement_name` | [`text`]                     | The name given by the client library to the prepared statement.                                                                                                                                                                                                               |
+| `session_id`              | [`uuid`]                     | An ID that is unique for each session.                                                                                                                                                                                                                                        |
```

Comment:
Does this join with `id` in `mz_internal.mz_sessions`?

Reply:
No, because those IDs get reused, so I had to invent a new globally unique one. It _does_ join against mz_session_history, but we don't (yet) document that.